SrfGen/createSourceRealisation.py

```diff
@@ -2,11 +2,16 @@
 
```

Comment:
I know this isn't your script, but you probably have a good idea of what it does, so maybe add a short overview as to what it does/used for.